zds/member/views.py

```diff
@@ -139,6 +139,15 @@ def unregister(request):
                 article.authors.add(external)
             article.authors.remove(current)
             article.save()
+    # comments likes / dislikes
+    for like in CommentLike.objects.filter(user=current):
+        like.comments.like -= 1
+        like.comments.save()
+        like.delete()
+    for dislike in CommentDislike.objects.filter(user=current):
+        dislike.comments.dislike -= 1
+        dislike.comments.save()
+        dislike.delete()
```

Comment:
Je confirme. Regarder les sujets suivants peut-être : 

* https://docs.djangoproject.com/en/1.6/ref/models/querysets/#select-for-update

Je tenterai un
```
likes = CommentLike.objects.select_for_update().filter(user=current).update(likes=F('like')-1).delete()
```

si ça fonctionne le update + delete dans le même query